README.rst

```diff
@@ -2,10 +2,29 @@ Transaction Uploader
 ====================
 
 Reads Money to Prisoners (MTP) bank account to facilitate reconciliation.
-Part of the Prisoner Money suite of apps.
+Part of the `Prisoner Money suite of apps`_.
+
+This app does not have a web interface. It's run as a cron-like jon.
+
+Requirements
+------------
+
+- Python 3.6+
 
 Usage
 -----
+
+It's recommended that you use a python virtual environment to isolate each application.
+Please call this ``venv`` and make sure it's in the root folder of this application so that
```

Comment:
Just adding a note the comments on https://github.com/ministryofjustice/money-to-prisoners-api/pull/600 probably apply here too (only if anything changes of course).